spec/integration/plugins/timestamp_spec.rb

```diff
@@ -0,0 +1,11 @@
+RSpec.describe 'Plugins / :timestamps' do
+  include_context 'relations'
+
+  with_adapters do
+    context "with Create command" do
+
+    end
+
+  end
+
+end
```

Comment:
Er, yep.

I'm not actually sure about the split between Unit an Integration in these cases; I ended up following one of the _other_ plugin specs and building into unit, but ... I tend to feel like things that actually hit the database a Really Integration Tests.  

:shrug: